meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in

```diff
@@ -131,40 +131,63 @@ fi
 [ -z "${boardType}" ] && boardType="gp"
 [ -z "${devID}" ] && devID="$(lsusb | grep 'Texas Instruments, Inc. AM62x' | awk '{print $6}')"
```

Comment:
on my EVM the lsusb is "Texas Instruments, Inc. USB2.0 Hub" or "Texas Instruments, Inc. USB2.1 Hub" on a usb3 hub

Reply:
Changed to `Texas Instruments, Inc.`